lib/src/flex/Flex.stories.tsx

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import Title from "../../.storybook/components/Title";
+import styled from "styled-components";
+import DxcFlex from "./Flex";
+
+export default {
+  title: "Flex",
+  component: DxcFlex,
+};
+
+export const Chromatic = () => (
+  <>
+    <Title title="Default" level={4} />
+    <Container>
+      <DxcFlex>
+        <Placeholder />
+        <Placeholder width="50px" />
+        <Placeholder />
+        <Placeholder width="50px" />
+        <Placeholder width="50px" />
+      </DxcFlex>
+    </Container>
+    <Title title="Direction column, wrap, justify content end, align items center and gap" level={4} />
+    <Container>
+      <DxcFlex direction="column" wrap="wrap" justifyContent="end" alignItems="center" gap="20px">
+        <Placeholder />
+        <Placeholder width="100px" />
+        <Placeholder />
+        <Placeholder width="100px" />
+        <Placeholder />
+      </DxcFlex>
+    </Container>
+    <Title title="Wrap with align content space between, row and column gaps, and as a span" level={4} />
+    <Container height="250px">
+      <DxcFlex wrap="wrap" alignContent="space-between" as="span" gap={{ rowGap: "10px", columnGap: "20px" }}>
+        <Placeholder />
+        <Placeholder />
+        <Placeholder />
+        <Placeholder />
+        <Placeholder width="100px" />
+        <Placeholder />
+        <Placeholder />
+        <Placeholder width="100px" />
+        <Placeholder />
+        <Placeholder />
+        <Placeholder width="100px" />
+        <Placeholder />
+      </DxcFlex>
+    </Container>
+    <Title title="Basis 100%, order, grow and align self" level={4} />
+    <Container height="75px">
+      <DxcFlex basis="100%">
+        <DxcFlex order={3} grow={1} alignSelf="flex-end">
+          <PlaceholderGrowAndShrink>order 3, grow 1, align self end</PlaceholderGrowAndShrink>
+        </DxcFlex>
+        <DxcFlex order={-1} grow={4}>
+          <PlaceholderGrowAndShrink>order -1, grow 4</PlaceholderGrowAndShrink>
+        </DxcFlex>
+        <DxcFlex order={5} grow={1}>
+          <PlaceholderGrowAndShrink>order 5, grow 1</PlaceholderGrowAndShrink>
+        </DxcFlex>
+        <DxcFlex order={2} grow={2}>
+          <PlaceholderGrowAndShrink>order 2. grow 2</PlaceholderGrowAndShrink>
+        </DxcFlex>
+      </DxcFlex>
+    </Container>
+    <Title title="Basis and shrink" level={4} />
```

Comment:
This example doesn't show the effect of `flex-shrink`. I suggest changing this example quite a bit:

```
      <DxcFlex basis="500px" gap="20px">
        <DxcFlex grow={1} shrink={2} basis="400px">
          <PlaceholderGrowAndShrink>This item shrinks</PlaceholderGrowAndShrink>
        </DxcFlex>
        <DxcFlex grow={1} basis="400px">
          <PlaceholderGrowAndShrink>Item 2</PlaceholderGrowAndShrink>
        </DxcFlex>
        <DxcFlex grow={1} basis="400px">
          <PlaceholderGrowAndShrink>Item 3</PlaceholderGrowAndShrink>
        </DxcFlex>
      </DxcFlex>
```

![image](https://user-images.githubusercontent.com/44321109/182562762-bc8a5f83-0ace-432f-9b6f-b440ea235433.png)

Reply:
Done